var/spack/repos/builtin/packages/spades/package.py

```diff
@@ -13,6 +13,7 @@ class Spades(CMakePackage):
     homepage = "http://cab.spbu.ru/software/spades/"
     url      = "http://cab.spbu.ru/files/release3.10.1/SPAdes-3.10.1.tar.gz"
 
+    version('3.13.0', git='https://github.com/ablab/spades.git', tag='v3.13.0')
```

Comment:
Why switch to git? Also, you can add `git` as a top-level parameter like `url` now.